pre/remap_restart/remap_command_line.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env python3
+#
+# source install/bin/g5_modules
+#
+# Newer GEOS code should load a module with GEOSpyD Python3 if not run:
+#   module load python/GEOSpyD/Min4.10.3_py3.9
+#
+#  This script parses and converts the command-line arguments to answers equivalent to that from remap_questions.py
+
+
+import os
+import subprocess
+import shlex
+import ruamel.yaml
+import shutil
+import questionary
+import glob
+import argparse
+from remap_utils import *
+from remap_params import * 
+
+def parse_args(program_description):
+
+    parser = argparse.ArgumentParser(description='Remap restarts',epilog=program_description,formatter_class=argparse.RawDescriptionHelpFormatter)
+
+    p_sub = parser.add_subparsers(help = 'Sub command help')
+    p_config  = p_sub.add_parser(
+     'config_file',
+     help = "Use config file as input",
+    )
+
+    p_config.add_argument('-c', '--config_file',  help='YAML config file')
+
+    p_command = p_sub.add_parser(
+     'command_line',
+     help = "Use command line as input",
+    )
+    p_command.add_argument('-merra2', action='store_true', default= False, help='use merra2 restarts')
+    p_command.add_argument('-ymdh',              help='yyyymmddhh year month date hour of input and new restarts')
+    p_command.add_argument('-grout',             help='Grid ID of new restarts, format Cxx')
+    p_command.add_argument('-levsout',           help='Levels for new restarts')
+
+    p_command.add_argument('-out_dir',           help='Directory for new restarts')
+    p_command.add_argument('-rst_dir',           help='Directory of input restarts')
+
+    p_command.add_argument('-expid',             help='Experiment ID of input restarts')
+    p_command.add_argument('-newid', default="", help='Experiment ID for new restarts')
+
+    p_command.add_argument('-bcvin',             help='Boundary conditions version of input restarts')
+    p_command.add_argument('-bcvout',            help='Boundary conditions version for new restarts')
+
+    p_command.add_argument('-in_wemin',          help='Min. water snow water equivalent param. used with input restarts')
+    p_command.add_argument('-out_wemin',         help='Min. water snow water equivalent param. to be used with new restarts')
+
+    ocean_grids=['360x180','1440x720','2880x1440','CS', '72x36', '360x200','720x410','1440x1080']
```

Comment:
@weiyuan-jiang : There is some duplication in the "choices" definitions between this script and remap_questions.py.  Can this be done in just one place and used in both scripts?  Also for `data model choices`, `coupled model choices`, and `['data', 'MOM5', 'MOM6']` below, and possibly other lists.

Reply:
Yes, it can be done

Reply:
Thanks, @weiyuan-jiang, for avoiding the definition of the same lists multiple times **within** the `remap_command_line.py` script. This is better.
However, my original question went farther.  Can we use the lists defined in `remap_questions.py` (e.g., "choices_omodel", "choices_ogrid_data", "choices_ogrid_cpld", ...) to assemble the "choices" in `remap_command_line.py`?  That is, avoid hardwiring the same lists in two different scripts.
If this doesn't work, could we avoid defining lists in `remap_command_line.py` altogether and use "help" messages such as "see remap_questions.py for a list of choices"?